ios/RNSelectableTextView.m

```diff
@@ -1,8 +1,22 @@
-
+#if __has_include(<RCTText/RCTTextSelection.h>)
```

Comment:
was this file generated? if so, how did you do it?

Reply:
No it was not auto generated, I just had a look at how e.g. react-native-sentry solves this issue and copied the pattern.